diff -Naur Boost/libs/locale/build/Jamfile.v2 Boost_patch/libs/locale/build/Jamfile.v2
--- Boost/tools/build/src/tools/clang-darwin.jam	2017-04-17 04:22:26.000000000 +0200
+++ Boost_patch/tools/build/src/tools/clang-darwin.jam	2017-06-15 07:55:47.000000000 +0200
@@ -169,6 +169,10 @@

 flags clang-darwin.link USER_OPTIONS <linkflags> ;

+# Set the max header padding to allow renaming of libs for installation.
+flags clang-darwin.link.dll OPTIONS : -headerpad_max_install_names ;
+
+
 # Declare actions for linking
 rule link ( targets * : sources * : properties * )
 {
